packages/dai-plugin-mcd/test/schemas/cat.spec.js

```diff
@@ -0,0 +1,44 @@
+import { mcdMaker } from '../helpers';
+import { takeSnapshot, restoreSnapshot } from '@makerdao/test-helpers';
+import BigNumber from 'bignumber.js';
+
+import schemas, {
+  LIQUIDATOR_ADDRESS,
+  LIQUIDATION_PENALTY,
+  MAX_AUCTION_LOT_SIZE
+} from '../../src/schemas';
+
+let maker, snapshotData;
+
+beforeAll(async () => {
+  maker = await mcdMaker({
+    multicall: true
+  });
+
+  snapshotData = await takeSnapshot(maker);
+  maker.service('multicall').createWatcher({ interval: 'block' });
+  maker.service('multicall').registerSchemas(schemas);
+  maker.service('multicall').start();
+});
+
+afterAll(async () => {
+  await restoreSnapshot(snapshotData, maker);
+});
+
+test(LIQUIDATOR_ADDRESS, async () => {
+  const expected = '0x55320248dC50Ef6dABc88ECbc294Fd5e2e1f4eC6';
```

Comment:
don't hardcode this -- import it from testnet.json instead

Reply:
please do the same with any other hardcoded addresses in the tests

Reply:
Magic addresses removed from tests.